eng/pipelines/coreclr/templates/build-corelib-job.yml

```diff
@@ -0,0 +1,163 @@
+
+    gatherAssetManifests: true
+    variables:
+    - name: osGroup
+      value: ${{ parameters.osGroup }}
+    - name: osSubgroup
+      value: ${{ parameters.osSubgroup }}
+    - name: stripSymbolsArg
+      value: ''
+    # Strip symbols only on the release build
+    - ${{ if eq(parameters.buildConfig, 'Release') }}:
+      - name: stripSymbolsArg
+        value: '-stripsymbols'
+    - name: compilerArg
+      value: ''
+    - name: publishLogsArtifactPrefix
+      value: 'BuildLogs_CoreCLR'
+    - ${{ if eq(parameters.compilerName, 'gcc') }}:
+      - name: compilerArg
+        value: '-gcc'
+      - name: publishLogsArtifactPrefix
+        value: 'BuildLogs_CoreCLR_GCC'
+      # workaround for gcc directory not in PATH
+      - name: CLR_CC
+        value: /opt/rh/devtoolset-7/root/usr/bin/gcc
+      - name: CLR_CXX
+        value: /opt/rh/devtoolset-7/root/usr/bin/g++
+    - ${{ if and(ne(parameters.osGroup, 'Windows_NT'), eq(parameters.compilerName, 'clang')) }}:
+      - name: compilerArg
+        value: '-clang9'
+      # Our FreeBSD doesn't yet detect available clang versions, so pass it explicitly.
+      - ${{ if eq(parameters.osGroup, 'FreeBSD') }}:
+        - name: compilerArg
+          value: '-clang6.0'
+      # Building for x64 MUSL happens on Alpine Linux and we need to use the stable version available there
+      - ${{ if and(eq(parameters.osGroup, 'Linux'), eq(parameters.osSubgroup, '_musl'), eq(parameters.archType, 'x64')) }}:
+        - name: compilerArg
+          value: ''
+      # AppleClang has different version scheme, so we let complier introspection pick up the available clang from PATH
+      - ${{ if eq(parameters.osGroup, 'OSX') }}:
+        - name: compilerArg
+          value: ''
+    - ${{ if and(ne(variables['System.TeamProject'], 'public'), ne(variables['Build.Reason'], 'PullRequest')) }}:
+      # Variables used to publish packages to blob feed
+      - name: dotnetfeedUrl
+        value: https://dotnetfeed.blob.core.windows.net/dotnet-coreclr/index.json
+      - name: dotnetfeedPAT
+        value: $(dotnetfeed-storage-access-key-1)
+      # Variables used by arcade to gather asset manifests
+      - name: _DotNetPublishToBlobFeed
+        value: true
+    - name: officialBuildIdArg
+      value: ''
+    - ${{ if and(eq(variables['System.TeamProject'], 'internal'), ne(variables['Build.Reason'], 'PullRequest')) }}:
+      - name: officialBuildIdArg
+        value: '/p:OfficialBuildId=$(Build.BuildNumber)'
+    - name: enforcePgoArg
+      value: ''
+    # The EnforcePGO script is only supported on Windows and is not supported on arm or arm64.
+    - ${{ if and(eq(parameters.buildConfig, 'Release'), and(eq(parameters.osGroup, 'Windows_NT'), not(or(eq(parameters.archType, 'arm64'), eq(parameters.archType, 'arm'))))) }}:
+      - name: enforcePgoArg
+        value: '-enforcepgo'
+
+    - ${{ parameters.variables }}
+
+    steps:
+
+    # Install native dependencies
```

Comment:
native dependencies might not be required.

Reply:
We need to make sure python is installed, so we probably still need this.